### PR TITLE
Docs landing page

### PR DESCRIPTION
**Pull request by plaindocs**
Looking at highlighting some different facets of the docs. These need some design work. CC @scopegate

### CODE REVIEW

**Comment by scopegate** [comment]
@plaindocs the first take on a new landing page is now in place — I've still gotta put a bunch of content in and check on the way I've implemented the layout and controller, but it's a start 👍🏻

**Comment by plaindocs** [comment]
Awesomeness!

**Comment by plaindocs** [comment]
This is looking great!

image

I reckon we should probably just chop down the install list, as well as removing the second intro line "Install agents on:" Maybe

- Ubuntu
- macOS
- Windows
- AWS
- Google Cloud
- More…
- SSH keys for private repositories

And I think we might need to highlight the tag line differently/

**Comment by scopegate** [comment]
image

Ready for a review now, I think! Other than a general review for code quality and any design suggestions, I'd really appreciate some input on:

- Ensuring that I haven't broken the Algolia search biz on the homepage
- A check that what I've done to the Rails controllers makes sense
- Sanity check on content (ping @plaindocs)

**Comment by plaindocs** [comment]
This looks awesome! :tada: :star: 

- [ ] I've rebooted the preview, not :100: sure if that will let us check the Algolia search box due to the URL.
- [ ] I'll give the content a final edit tomorrow. 
- [ ] Do we want a generic "more..." link for each section? 
- [ ] I'm wondering if that "Popular articles" is too much as it is?

**Comment by plaindocs** [comment]
Looks like the images are at `images/docs/icons/` but referenced at   `/docs/assets/icons/`

File: app/views/pages/index.html.erb [hunk 1]
```diff
@@ -1,29 +1,29 @@
 <div class="HomeSuggestions">
-<strong>📊 Popular Articles:</strong> 
-  <%= link_to "Getting Started", "/docs/tutorials/getting-started" %>, 
-  <%= link_to "Defining Build Steps", "/docs/pipelines/defining-steps" %>,
-  <%= link_to "Command Steps", "/docs/pipelines/command-step" %>, 
-  <%= link_to "Environment Variables", "/docs/pipelines/environment-variables" %>
+<strong>📊 Popular articles:</strong>
+  <%= link_to "Getting started", "/docs/tutorials/getting-started" %>,
+  <%= link_to "Defining build steps", "/docs/pipelines/defining-steps" %>,
+  <%= link_to "Command steps", "/docs/pipelines/command-step" %>,jjjjjjjjjjj
```
Comment (JuanitoFatas) [inline app/views/pages/index.html.erb]: `jjjjjjjjjjj` 🤣

**Comment by nikwek** [comment]
image

File: config/routes.rb [hunk 2]
```diff
@@ -115,11 +115,14 @@
   # Quick Reference JSON
   get "/docs/quick-reference/pipelines", to: "quick_reference#pipelines", as: :pipelines_quick_reference
 
+  # Homepage
+  get "/docs/index" => "pages#index", as: :home_page
```
Comment (blaknite) [inline config/routes.rb:119]: Any reason for having the explicit `/docs/index` instead of just `/docs`?

File: pages/index.md.erb [hunk 3]
```diff
@@ -0,0 +1,64 @@
+# Landing pages
```
Comment (blaknite) [inline pages/index.md.erb:1]: Is this file required?

Comment (plaindocs) [reply pages/index.md.erb:1]: Oh good catch, that is where I started mocking out the content, will delete.

**Comment by plaindocs** [comment]
- [x] Looks like the images are at `images/docs/icons/` but referenced at `/docs/assets/icons/`

OK, after quite some futzing looks like I managed to fix this in bec3440

**Comment by nikwek** [comment]
👍

**Review by nikwek — APPROVED** [review]

File: config/routes.rb (hunk 2 above)
Comment (JuanitoFatas) [reply config/routes.rb]: Fixed in https://github.com/buildkite/docs/pull/841/commits/45cee9bbee55e9c2cb458eae91b13d8bb46ed24f

**Review by JuanitoFatas — APPROVED** [review]
I reckon this is good to go and we can improve things further later.

**Comment by JuanitoFatas** [comment]
Awesome work @scopegate @plaindocs !!!
